frontends/web/src/locales/en/app.json

```diff
@@ -654,6 +654,7 @@
     },
     "button": "Sign and Send",
     "coincontrol": {
+      "title": "Send from output",
```

Comment:
I haven't studied where this is used but if it says `title` it should probably be in title-case (with a capital `O` in output).